libs/marshalling/zk/example/CMakeLists.txt

```diff
@@ -15,15 +15,11 @@ macro(define_marshalling_example name)
     add_executable(marshalling_${name}_example ${name}.cpp)
 
     target_link_libraries(marshalling_${name}_example PRIVATE
-
                           crypto3::multiprecision
                           crypto3::algebra
                           crypto3::zk
                           ${CMAKE_WORKSPACE_NAME}::marshalling-multiprecision
                           ${CMAKE_WORKSPACE_NAME}::marshalling-algebra
-                          ${CMAKE_WORKSPACE_NAME}::crypto3_zk
```

Comment:
Our examples don't compile anyway right? We can still  remove these, but that will not help right?

Reply:
At least cmake won't  complain if the project is built with `EXAMPLES_ENABLED=TRUE`